crates/sui-indexer/migrations/2022-12-01-034426_objects/up.sql

```diff
@@ -69,53 +69,53 @@ CREATE INDEX objects_history_old_owner_index ON objects_history (old_owner_type,
 CREATE TABLE objects_history_fast_path_partition_0 PARTITION OF objects_history FOR VALUES FROM (-1) TO (0);
 CREATE TABLE objects_history_partition_0 PARTITION OF objects_history FOR VALUES FROM (0) TO (MAXVALUE);
 
--- CREATE OR REPLACE FUNCTION objects_modified_func() RETURNS TRIGGER AS
--- $body$
--- BEGIN
---     IF (TG_OP = 'INSERT') THEN
---         INSERT INTO objects_history
---         VALUES (NEW.epoch, NEW.checkpoint, NEW.object_id, NEW.version, NEW.object_digest, NEW.owner_type,
---                 NEW.owner_address, NULL, NULL,
---                 NEW.initial_shared_version,
---                 NEW.previous_transaction, NEW.object_type, NEW.object_status, NEW.has_public_transfer,
---                 NEW.storage_rebate, NEW.bcs);
---         RETURN NEW;
---     ELSEIF (TG_OP = 'UPDATE') THEN
---         INSERT INTO objects_history
---         VALUES (NEW.epoch, NEW.checkpoint, NEW.object_id, NEW.version, NEW.object_digest, NEW.owner_type,
---                 NEW.owner_address, OLD.owner_type, OLD.owner_address,
---                 NEW.initial_shared_version,
---                 NEW.previous_transaction, NEW.object_type, NEW.object_status, NEW.has_public_transfer,
---                 NEW.storage_rebate, NEW.bcs);
---         -- MUSTFIX(gegaowp): we cannot update checkpoint in-place, b/c checkpoint is a partition key,
---         -- we need to prune old data in this partition periodically, like pruning old epochs upon new epoch.
---         RETURN NEW;
---     ELSIF (TG_OP = 'DELETE') THEN
---         -- object deleted from the main table, archive the history for that object
---         DELETE FROM objects_history WHERE object_id = old.object_id;
---         RETURN OLD;
---     ELSE
---         RAISE WARNING '[OBJECTS_MODIFIED_FUNC] - Other action occurred: %, at %',TG_OP,NOW();
---         RETURN NULL;
---     END IF;
+CREATE OR REPLACE FUNCTION objects_modified_func() RETURNS TRIGGER AS
+$body$
+BEGIN
+    IF (TG_OP = 'INSERT') THEN
+        INSERT INTO objects_history
+        VALUES (NEW.epoch, NEW.checkpoint, NEW.object_id, NEW.version, NEW.object_digest, NEW.owner_type,
+                NEW.owner_address, NULL, NULL,
+                NEW.initial_shared_version,
+                NEW.previous_transaction, NEW.object_type, NEW.object_status, NEW.has_public_transfer,
+                NEW.storage_rebate, NEW.bcs);
+        RETURN NEW;
+    ELSEIF (TG_OP = 'UPDATE') THEN
+        INSERT INTO objects_history
+        VALUES (NEW.epoch, NEW.checkpoint, NEW.object_id, NEW.version, NEW.object_digest, NEW.owner_type,
+                NEW.owner_address, OLD.owner_type, OLD.owner_address,
+                NEW.initial_shared_version,
+                NEW.previous_transaction, NEW.object_type, NEW.object_status, NEW.has_public_transfer,
+                NEW.storage_rebate, NEW.bcs);
+        -- MUSTFIX(gegaowp): we cannot update checkpoint in-place, b/c checkpoint is a partition key,
```

Comment:
feel free to ignore this, this is related to fast path on indexer, which is disabled and pro no longer needed based on our convo about read-after-write-consistency